internal/provider/postgres/resource/validate.go

```diff
@@ -17,12 +19,12 @@ func ValidatePostgresPlanFunc() validator.String {
 
 func isNonCustomPostgresPlanFunc() validator.String {
 	return stringvalidator.OneOf(
-		string(client.PostgresPlansFree),
-		string(client.PostgresPlansStarter),
-		string(client.PostgresPlansStandard),
-		string(client.PostgresPlansPro),
-		string(client.PostgresPlansProPlus),
-		string(client.PostgresPlansCustom),
+		string(postgres.Free),
+		string(postgres.Starter),
+		string(postgres.Standard),
+		string(postgres.Pro),
+		string(postgres.ProPlus),
+		string(postgres.Custom),
```

Comment:
Not added introduced in this PR, but it seems like `Custom` shouldn't be in this list based on the function name.